routers/web/repo/compare.go

```diff
@@ -567,6 +567,15 @@ func PrepareCompareDiff(
 	} else {
 		title = headBranch
 	}
+	if len(title) > 255 {
+		if ctx.Data["content"] != nil {
+			ctx.Data["content"] = fmt.Sprintf("...%s\n\n%s", title[252:], ctx.Data["content"])
```

Comment:
Could use Unicode character instead of "..." and use 254 characters for title

Reply:
So I initially considered that but remembered - perhaps incorrectly - that github doesn't use … `U+2026` for some reason itself.

Reply:
We use in all UI parts it already

Reply:
I mean if we're willing to risk the mysql and mssql users' wrath and/or people who insist on non utf8 in their commit logs then switching to use ellipsis instead would be easy.

Reply:
I don't understand this to be honest 😉... Not that there is still software that does not support utf8
Anyway this was just suggestion and is non blocking from my side

Reply:
lets try the ellipsis.